docs/content/documentation/getting-started/installation.md

```diff
@@ -65,7 +65,7 @@ download previous versions of Gutenberg on your package manager of choice in the
 Gutenberg is available on [Brew](https://brew.sh):
 
 ```bash
-$ brew install gutenberg
+$ brew install zola
```

Comment:
This section is for gutenberg, can you cut and paste the `macOS` subsection and move it above in the Zola section?